user/languages/go.md

```diff
@@ -43,11 +55,12 @@ go:
 
 All go version management is handled by [gimme](https://github.com/travis-ci/gimme).
 
-For precise versions pre-installed on the VM, please consult "Build system information" in the build log.
-
 ## Go Import Path
 
-The project source code will be placed in `GOPATH/src/github.com/user/repo` by default, but if [vanity imports](https://golang.org/cmd/go/#hdr-Remote_import_paths) are necessary (especially for [`internal` package imports](https://golang.org/cmd/go/#hdr-Internal_Directories)), `go_import_path:` may be specified at the top level of the config, e.g.:
+The project source code will be placed in `GOPATH/src/github.com/user/repo` by
```

Comment:
Does this mean anything code is copied there? Or that it should be placed there? Or ?